### DISCUSSION TITLE
Why the time of a quarter in default tempo is 480?

### DISCUSSION THREAD

**Discussion by alawab** [mido/mido#378] [Q&A]
Hi,

According to the definition of tempo in midi, tempo 500000ms per beat = bpm 120, which means a quarter note should be 500ms, however, actually it's 480 when I create a midi file. Why? and what's the correct calculation?

**Comment by rdoursenaud** [comment]
I think you are conflating two different concepts: `Tempo` and `MIDI ticks`.

`MIDI ticks` are the internal representation of time.
The [specification](https://www.midi.org/specifications/file-format-specifications/standard-midi-files/rp-001-v1-0-standard-midi-files-specification-96-1-4-pdf) details `Ticks per quarter note` which is the resolution of this time representation.
For some reason, `mido` implemented it as `Ticks per beat`.

`480` is the number of `Ticks per beat` which translates to `120` `Ticks per quarter note` in 4/4.

The tempo itself is specified in microseconds per quarter note and assumed to be 500 000 µS (aka 120 BPM in 4/4) if not specified.

**Reply by rdoursenaud** [reply]
P.S.: There is a known [issue](https://github.com/mido/mido/issues/102) with a [proposed fix](https://github.com/mido/mido/pull/114) relative to the conversions between these units that you may find useful.
